Dockerfile

```diff
@@ -0,0 +1,22 @@
+ARG PYTHON_VERSION=3.8
+
+FROM python:${PYTHON_VERSION} AS build
+
+WORKDIR /app
+
+COPY . /app
+
+FROM python:${PYTHON_VERSION} AS run
+
+WORKDIR /app
+
+ENV PYTHONUNBUFFERED=1
+
+COPY --from=build /app .
+
+RUN pip install -r requirements.txt
+RUN python manage.py migrate
```

Comment:
The 'python manage.py migrate' command assumes that Django is being used. Ensure that Django is installed and that the manage.py file exists in the working directory.